tiles/tall_grass.gd

```diff
@@ -26,3 +26,8 @@ func process_tile(tile):
 	var grass_cut = preload("res://effects/grass_cut.tscn").instance()
 	network.current_map.add_child(grass_cut)
 	grass_cut.global_position = map_to_world(tile) + Vector2(8,6)
+	
```

Comment:
What's the difference between ` tiles./tall_grass.gd` and `tiles/tall_grass.gd`?

It looks like you make a duplicate.

Reply:
Yep, duplicate is deleted now